app/unionFind.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Created by tatsiak on 4/1/17.
+ */
+ 'use strict'
+ class UnionFind {
+    constructor() {
+        this._elements = [];
+    }
+
+    connect(a, b) {
+        if (this.isConnected(a, b)) return;
+        if (isNaN(this._elements[a])) {
+         this._elements[a] = a;
+             // this._elements[1]=1;
+         }
+         this._elements[b] = isNaN(this._elements[b]) ? b : this._elements[b];
```

Comment:
in this case it was useful to use a ternary operator
Is there some rules about using only "if else" or only ternary operators?